### PR TITLE
Recognize contributor and fix typo in generated HTML page

### PR DESCRIPTION
**Pull request by big-guy**
replaces https://github.com/gradle/gradle/pull/26669

### Reviewing cheatsheet

Before merging the PR, comments starting with 
- ❌ ❓**must** be fixed
- 🤔 💅 **should** be fixed
- 💭 **may** be fixed
- 🎉 celebrate happy things

### CODE REVIEW

**Review by lkasso — APPROVED** [review]

**Comment by big-guy** [comment]
@bot-gradle test this

File: platforms/software/testing-base/src/main/java/org/gradle/api/internal/tasks/testing/report/PackagePageRenderer.java [hunk 1]
```diff
@@ -32,7 +32,7 @@ protected void renderBreadcrumbs(SimpleHtmlWriter htmlWriter) throws IOException
 
     private void renderClasses(SimpleHtmlWriter htmlWriter) throws IOException {
         htmlWriter.startElement("table");
-        htmlWriter.startElement("thread");
+        htmlWriter.startElement("thead");
```
Comment (big-guy) [inline platforms/software/testing-base/src/main/java/org/gradle/api/internal/tasks/testing/report/PackagePageRenderer.java:35]: AFAICT, this had no impact on the rendered page.

Comment (octylFractal) [reply platforms/software/testing-base/src/main/java/org/gradle/api/internal/tasks/testing/report/PackagePageRenderer.java:35]: As we don't render a `tbody`, yes, this element effectively did nothing.

File: subprojects/core/src/main/java/org/gradle/internal/html/SimpleHtmlWriter.java [hunk 2]
```diff
@@ -38,4 +41,30 @@ public SimpleHtmlWriter(Writer writer, String indent) throws IOException {
...
+    public SimpleMarkupWriter startElement(String name) throws IOException {
+        if (!isValidHtmlTag(name)) {
+            throw new IllegalArgumentException(String.format("Invalid HTML tag: '%s'", name));
+        }
+        return super.startElement(name);
+    }
+
+    // All valid tags should be in lowercase
+    // Add more tags as necessary
+    private final static List<String> VALID_HTML_TAGS = Arrays.asList(
```
Comment (octylFractal) [inline subprojects/core/src/main/java/org/gradle/internal/html/SimpleHtmlWriter.java:55]: 🤔 Can this be a `Set` instead?

Comment (big-guy) [reply subprojects/core/src/main/java/org/gradle/internal/html/SimpleHtmlWriter.java]: done

**Comment by big-guy** [comment]
@bot-gradle test and merge

**Comment by big-guy** [comment]
@bot-gradle test and merge

**Pull request merged**